app/Controllers/TaskController.php

```diff
@@ -64,67 +61,44 @@ public function create(): void
             $this->task->setNotificationSent(1);
 
             $inserted = $this->task->save();
-            $alertMessage = ($inserted) ? "<div class='alert alert-success' role='alert'>Task added successfully!</div>" :
-                        "<div class='alert alert-danger' role='alert'>Something went wrong!</div>";
+            $alertMessage = ($inserted) ? "<div class='alert alert-success' role='alert'>Task added successfully!</div>" : "<div class='alert alert-danger' role='alert'>Something went wrong!</div>";
 
         }
         $this->render('addTask', ["alertMessage" => $alertMessage, "categories" => $categories]);
 
     }
 
     /**
-     * @throws DateMalformedStringException
-     * @throws \PHPMailer\PHPMailer\Exception
+     * @return void
      */
-    public function index(): void
+    public function extracted(): void
```

Comment:
extracted in not a good name for method this one is generated by PhpStorm but you need to give it a meaningful name though